packages/usa-input-mask/src/content/usa-input-mask-alphanumeric.json

```diff
@@ -1,10 +1,11 @@
 {
-  "label": "Alphanumeric",
-  "hint": "For example, A1B 2C3",
+  "forceupper": "true",
```

Comment:
Small change, but I found the `forceupper` text field control to be a bit clunky in Storybook. If we remove the quotes, the control will appear as a boolean toggle instead of a text field, which seems easier to control.

```suggestion
  "forceupper": true,
```

<img width="470" alt="image" src="https://user-images.githubusercontent.com/93996430/214109958-bd7c271d-df6f-47a7-a6f6-68bd3fb26537.png">
<img width="553" alt="image" src="https://user-images.githubusercontent.com/93996430/214110222-cbf3a231-476d-46f1-acef-51dc84e80402.png">